js/tools/bezier.js

```diff
@@ -0,0 +1,56 @@
+/* global C2D */
+window.tools.bezier = {
+    name: 'bezier',
+    icon: '/images/bezier.png',
+    state: {
+      selected: false,
+      mousePressed: false,
+      prevMouse: { x: null, y: null }
+    },
+  
+    events: {
+      mousedown: function () {
+        const state = window.tools.bezier.state
+        state.mousePressed = true
+      },
+      mouseup: function () {
+        const state = window.tools.bezier.state
+        state.mousePressed = false
+        state.prevMouse = { x: null, y: null }
+
+      },
+      mousemove: function (e) {
+        const state = window.tools.bezier.state
+        if (state.selected && state.mousePressed) {
+          const mouse = C2D.eventToMouse(e)
+          const px = state.prevMouse.x || mouse.x
+          const py = state.prevMouse.y || mouse.y
+          const ctx = C2D.ctx
+          const cc = document.createElement('canvas')
+          cc.width = C2D.width/2
+          cc.height = C2D.height/2
```

Comment:
did u mean to create a new canvas here? or did u mean to use the one that's already in the app `C2D.canvas`